bindings/pydrake/multibody/plant_py.cc

```diff
@@ -130,11 +130,16 @@ PYBIND11_MODULE(plant, m) {
             [](Class * self,
                 std::unique_ptr<UniformGravityFieldElement<T>> force_element)
                 -> auto& {
-              // N.B. We need to make sure we call the correct specialization in
-              // MultibodyPlant for it to take note we are adding gravity to the
-              // model. This is ugly API needs to be updated, see #11080.
+              WarnDeprecated(
+                  "Use mutable_gravity_field.set_gravity_vector() instead.");
```

Comment:
this should be "mutable_gravity_field().set_gravity_vector()".  the warning i got in underactuated felt like it was unactionable, because I didn't know how to get the mutable_gravity_field.